xla/service/gpu/priority_fusion.cc

```diff
@@ -163,7 +163,10 @@ class GpuPriorityFusionQueue {
       }
       instructions.push_back(instruction);
     }
-
+    // write here and only read in worker threads
```

Comment:
Could you ellaborate in the comment what do we write here? It's not clear from the current comment.